docs/experiment/apis/management-api.md

```diff
@@ -478,13 +537,784 @@ Update the rollout weights for an experiment.
+PATCH https://management-api.experiment.amplitude.com/flags/{id}/variants/{variantKey}
+```
+
+Edit a variant for a flag.
+
+### Path variables
+
+|<div class="big-column">Name</div>|Description|
+|---|----|
+|`id`| Required. String. The flag's ID.|
+|`variantKey`| Required. String. The variant's ID.|
+
+### Request body
+
+|<div class="med-big-column">Name</div>|Requirement|Type|Description|
+|---|---|---|---|
+|`key`| Optional | string | The flag key. |
+|`description`| Optional | string | Description for the flag.|
+|`name`| Optional | string | Name for the flag.|
+|`payload`| Optional | string | Optional payload. Value must be a valid JSON element.|
+|`rolloutWeight`| Optional | number | Rollout weight for non-targeted users.|
+
+### Response
+
+A successful request returns a `200 OK` response.
+
+<!-- TODO example response body -->
+
+!!!example "Example cURL"
+    ```bash
+    curl --request PATCH \
+      --url 'https://management-api.experiment.amplitude.com/flags/<id>/variants/<variantKey>' \
+      --header 'Accept: application/json' \
+      --header 'Authorization: Bearer <management-api-key>'
+      --data '{"key":"<key>","name":"<name>","description":"<description>","payload":"<payload>","rolloutWeight":"<rolloutWeight>"}'
+    ```
+
+------
+
+## Remove flag variant
+
+```bash
+DELETE https://management-api.experiment.amplitude.com/flags/{id}/variants/{variantKey}
+```
+
+Remove a variant from an experiment.
+
+### Path variables
+
+|<div class="big-column">Name</div>|Description|
+|---|----|
+|`id`| Required. String. The flag's ID.|
+|`variantKey`| Required. String. The variant's ID.|
+
+### Response
+
+A successful request returns a `200 OK` response.
+
+<!-- TODO example response body -->
+
+!!!example "Example cURL"
+    ```bash
+    curl --request DELETE \
+      --url 'https://management-api.experiment.amplitude.com/flags/<id>/variants/<variantKey>' \
+      --header 'Accept: application/json' \
+      --header 'Authorization: Bearer <management-api-key>'
+    ```
+
+------
+
+## Add users to flag's variant
+
+```bash
+POST https://management-api.experiment.amplitude.com/flags/{id}/variants/{variantKey}/users
+```
+
+Add users (inclusions) to flag's variant.
+
+### Path variables
+
+|<div class="big-column">Name</div>|Description|
+|---|----|
+|`id`| Required. String. The flag's ID.|
+|`variantKey`| Required. String. The variant's ID.|
+
+### Request body
+
+|<div class="med-big-column">Name</div>|Requirement|Type|Description|
+|---|---|---|---|
+|`inclusions`| Required | string array | Array of user ids. |
+
+### Response
+
+A successful request returns a `200 OK` response.
+
+<!-- TODO example response body -->
+
+!!!example "Example cURL"
+    ```bash
+    curl --request POST \
+      --url 'https://management-api.experiment.amplitude.com/flags/<id>/variants/<variantKey>/users' \
+      --header 'Accept: application/json' \
+      --header 'Authorization: Bearer <management-api-key>'
+      --data '{"inclusions":"<[]>"}'
```

Comment:
This is a confusing structure. Is it a string or an array? The table above says array of strings but this seems to indicate a json array encoded as a string.

Reply:
I provided better explanation